### PR TITLE
Fix chunked Parquet reader benchmark

### PR DESCRIPTION
**Pull request by vuule**
## Description
Currently, chunked Parquet reader benchmark creates the chunked reader object once and reuses it for all iterations.
After the first iteration the source is fully read so each subsequent iteration returns a single, empty, chunk. 

This PR fixes the use of the chunked reader object.
The creation of the object is included in the benchmark timing.

## Checklist
- [x] I am familiar with the [Contributing Guidelines](https://github.com/rapidsai/cudf/blob/HEAD/CONTRIBUTING.md).
- [ ] New or existing tests cover these changes.
- [ ] The documentation is up to date with these changes.

### CODE REVIEW

**Review by ttnghia — APPROVED** [review]

File: cpp/benchmarks/io/parquet/parquet_reader_input.cpp [hunk 1]
```diff
@@ -149,8 +148,9 @@ void BM_parquet_read_chunks(
                try_drop_l3_cache();
 
                timer.start();
+               auto reader = cudf::io::chunked_parquet_reader(byte_limit, read_opts);
```
Comment (PointKernel) [inline cpp/benchmarks/io/parquet/parquet_reader_input.cpp:151]:
```suggestion
               auto reader = cudf::io::chunked_parquet_reader(byte_limit, read_opts);
               timer.start();
```
Do we want to move this line before `timer.start()` to exclude the reader construction time?

Comment (vuule) [reply cpp/benchmarks/io/parquet/parquet_reader_input.cpp:151]: I've thought about this and decided to include it. The reader object is not reusable, so IMO the time it takes to create it should be taken into account as part of the time needed to read a file. There's no way for the user to offset this cost.
In addition, the constructor might be doing a lot of preprocessing (e.g. to determine the chunks).
CC @nvdbaranec

Comment (PointKernel) [reply cpp/benchmarks/io/parquet/parquet_reader_input.cpp:151]: >  There's no way for the user to offset this cost.

good point

**Review by PointKernel — APPROVED** [review]
LGTM

**Pull request merged**